### PR TITLE
Add migration status logger

### PR DESCRIPTION
**Pull request by saisubodhMSFT**
Added migration status logs with the following fields

[MigrationLogUUID, SourceSiteName, MigrationStartTs, MigrationEndTs, TimeTakenInMs, MigrationStatus, FileContentSize, FilesCount, DBContentSize, Errors, SourceSiteWpVersion, SourceSitePhpVersion]

### CODE REVIEW

File: azure_app_service_migration/admin/engines/class-azure_app_service_migration-s2s-controller.php [hunk 1]
```diff
@@ -154,4 +162,38 @@ public static function s2s_cancel() {
...
 		wp_die();
 	}
+
+	private static function log_migration_status(Exception $e, $params) {
+		// Initialize migration status log fields
+		$body = array();
+		$body['migration_start_timestamp']= $params['migration_start_timestamp'];
+		$body['migration_end_timestamp'] = date('Y-m-d H:i:s', time());
+		$body['migration_time_taken'] = round(microtime(true) * 1000) - $params['migration_start_time'];
+		$body['source_website_name'] = getenv('WEBSITE_SITE_NAME');
```
Comment (ZubaeyrMSFT) [inline azure_app_service_migration/admin/engines/class-azure_app_service_migration-s2s-controller.php:172]: Source site is the external, the one from where data is getting transferred. Destination is app service.

File: azure_app_service_migration/admin/engines/common/class-azure_app_service_migration-enumerate-parser.php [hunk 2]
```diff
@@ -142,16 +142,12 @@ public static function send_files(&$params, $zip = null, $password = '' ) {
                 $relativePath = $row[2];
 
                 if (file_exists($filePath) && is_file($filePath)) {
-                    $temp_offset = 0;
```
Comment (ZubaeyrMSFT) [inline azure_app_service_migration/admin/engines/common/class-azure_app_service_migration-enumerate-parser.php:145]: Is this change related to migration log or some other bug fix?

Comment (saisubodhMSFT) [reply azure_app_service_migration/admin/engines/common/class-azure_app_service_migration-enumerate-parser.php:145]: This is a migration bug fix
